client/src/components/Ticketing/ticketingmanager/Events/showingInputContainer.tsx

```diff
@@ -7,36 +7,53 @@ import {Showing} from '../../../../interfaces/showing.interface';
  * Used to map props to input container correctly
  *
  * @module
- * @param {initialData} Showing
+ * @param {initialData} object
  * @param {number} id
- * @param {Function} addShow
- * @param {Function} deleteShow
+ * @param {Function} handleSetShow
  */
+interface InitialData {
+  availableseats?: number,
+  defaulttickettype?: number,
+  eventdate?: number | string,
+  eventid_fk?: number,
+  eventinstanceid?: number,
+  eventtime?: string,
+  ispreview?: boolean,
+  purchaseuri?: string,
+  salestatus?: boolean,
+  totalseats?: number
+}
+
```

Comment:
For some reason, the InitialData sends this object back instead of a Showings object. I want to look into this a little more, but this is a temporary fix for the type errors.

Reply:
Will this do the trick for the type error? 

![image](https://user-images.githubusercontent.com/25473863/224200451-063dae32-ea09-49b8-addd-20b19cad013e.png)